services/node_get_info.proto

```diff
@@ -0,0 +1,129 @@
+syntax = "proto3";
+
+package proto;
+
+/*
+ * Hedera Network Services Protobuf
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.addressbook">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+import "basic_types.proto";
+import "query_header.proto";
+import "response_header.proto";
+
+/**
+ * Gets information about Node instance. This needs super use privileges to succeed or should be a node operator.
+ */
+message NodeGetInfoQuery {
+    /**
+     * Standard information sent with every query operation.<br/>
+     * This includes the signed payment and what kind of response is requested
+     * (cost, state proof, both, or neither).
+     */
+    QueryHeader header = 1;
+
+    /**
+     * A node identifier for which information is requested.<br/>
+     * If the identified node is not valid, this request SHALL fail with
+     * a response code `INVALID_NODE_ID`.
+     */
+    uint64 node_id = 2;
+}
+
+/**
+ * A query response describing the current state of a node
+ */
+message NodeInfo {
+    /**
+     * A numeric node identifier.<br/>
+     * This value identifies this node within the network address book.
+     */
+    uint64 node_id = 1;
+
+    /**
+     * The account is charged for transactions submitted by the node that fail due diligence
+     */
+    AccountID account_id = 2;
+
+    /**
+     * A description of the node with UTF-8 encoding up to 100 bytes
+     */
+    string description = 3;
+
+    /**
+     * A node's Gossip Endpoints, ip address and port
+     */
+    repeated ServiceEndpoint gossip_endpoint = 4;
+
+    /**
+     * A node's service Endpoints, ip address or FQDN and port
+     */
+    repeated ServiceEndpoint service_endpoint = 5;
+
+    /**
+     * node’s public key within the Address Book used for signing.
+     * The node's X509 RSA public key used to sign stream files (e.g., record stream
+     * files).
+     */
+    bytes public_key = 6;
```

Comment:
Should this be modified to `gossip_ca_certificate` to match the recent change in create_node?